### DISCUSSION TITLE
mod Disable taskbar thumbnail preview Windhawk Mods

### DISCUSSION THREAD

**Discussion by Quesito35** [ramensoftware/windhawk-mods#307] [Q&A] [answered]
Can you add the mod add Disable taskbar thumbnail preview.?
Thank you.

**Comment by m417z** [comment]
It's planned. You can also vote for it here under **Hovering: List / Tooltip / Nothing**:
https://ramensoftware.com/windhawk-mods-for-the-windows-11-taskbar

**Reply by Quesito35** [reply]
I just voted.
Thank you very much.

**Reply by Anixx** [reply]
Will it affect only Win11 taskbar or both? I would like the 7+TT feature ported to Windhawk.

**Reply by m417z** [reply]
I'll try to make it work on both, but if the solution is substantially different for Win10 and Win11, I'll probably start from Win11.

**Comment by nazzy121** [comment]
Voted too.  Please let us disable thumbnail previews!

**Comment by VarunAgw** [comment]
I found this solution that also works

https://www.elevenforum.com/t/change-hover-time-to-show-taskbar-thumbnail-previews-in-windows-11.6366/

**Comment by m417z** [comment, marked as answer]
Done: [Disable Taskbar Thumbnails](https://windhawk.net/mods/taskbar-thumbnails).

**Reply by Quesito35** [reply]
Hi. 
I have installed the "Disable Taskbar Thumbnails" mod, but I still get the taskbar thumbnail preview.
Attached video.
Windows 11 Pro 22631.4317.
Any solution?.
Thanks.

https://github.com/user-attachments/assets/002e901f-2d0b-44a6-bf54-b1c20caf4d6c

**Reply by m417z** [reply]
What's your taskbar app? If it's StartAllBack then it's not supported, see https://github.com/ramensoftware/windhawk-mods/issues/844#issuecomment-2278122730.

**Reply by Quesito35** [reply]
StartAllback version 3.7.11
Should I uninstall it?

**Reply by m417z** [reply]
With StartAllBack older than 3.8, you can use the mod by enabling the "Customize the old taskbar on Windows 11" option in the settings.

**Reply by Quesito35** [reply]
Thank you very much. I followed the step you indicated (you can use the mod by enabling the "Customize the old taskbar in Windows 11" option in the settings) and now it works perfectly.
Solved.

https://github.com/user-attachments/assets/21469802-4651-40f1-8da2-57ac6412f812
